### PR TITLE
[19529] - update mpGraphQL 2.0 binaries to RC3 for 22.0.0.3 beta

### PR DESCRIPTION
**Pull request by WhiteCat22**
Update to RC3 binaries to certify TCK compliance with the beta feature released in 22.0.0.3.

### CODE REVIEW

File: dev/io.openliberty.microprofile.graphql.2.0.internal_fat_tck/publish/tckRunner/tck/pom.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- 
+<!--
     Copyright (c) 2021 IBM Corporation and others. All rights reserved.
```
Comment (jhanders34) [inline dev/io.openliberty.microprofile.graphql.2.0.internal_fat_tck/publish/tckRunner/tck/pom.xml]: Should the copyright update?

**Review by jhanders34 — APPROVED** [review]

**Comment by WhiteCat22** [comment]
#run-libby-bot

**Comment by WhiteCat22** [comment]
#build

**Review by jim-krueger — APPROVED** [review]
LGTM

**Pull request merged**